src/Avalonia.Controls/AppBuilder.cs

```diff
@@ -15,38 +16,53 @@ public abstract class AppBuilderBase<AppBuilder> where AppBuilder : AppBuilderBa
         /// <summary>
         /// Gets or sets the <see cref="IRuntimePlatform"/> instance.
         /// </summary>
-        public IRuntimePlatform RuntimePlatform { get; set; }
+        public IRuntimePlatform RuntimePlatform { get; private set; }
```

Comment:
Good point. I was just locking down this a little bit since I didn't really see a use case for changing it afterwards. I'll fix that (and the broken build).